src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@mirh The reason for my "AMD outburst" is simply that @Sami32 should know very well that I think life is too short to use things that you know isn't up to quality. I'm not saying that everything from AMD is bad, I'm just saying that my experience, both personal and professional, their quality standards aren't what I'd consider acceptable. That means that they will release things that I don't consider to be fully functional, so I have lost all trust in them and isn't willing to waste more time and money on their products only to risk being bitten.

That said I certainly remember some pretty noisy Nvidia cards with tiny fans running at very high RPMs. Its more the endless crashing/unstability, especially when it comes to CPU's - although I suspect that a lot of the blame was with the VIA chipsets as I remember having a few Intel/VIA motherboards as well suffering from similar unstability. But, as long as there's no other option (at least there wasn't the last time I used an AMD) than VIA, it doesn't really matter where the fault lies - you end up with an unstable product.

It's more an internal thing between me and @Sami32 though, I'm not saying that everybody else has to share my view, only that he should know my view by now 😉 It should also be said that I generally expect more reliability/stability than the average user that simply reinstalls their OS when things get too bad. I typically only reinstalls my OS when I for some reason is forced to upgrade, and I only upgrade when I have to. My C drive still has part of my file structure from my Windows 95 installation from 1997 in it - which means that I never actually format my OS drive but transfer its content when upgrading drive instead. Given this "use pattern", it's obviously more critical for me than many people that things aren't corrupted, which is a big risk with unstable hardware/drivers.